server.js

```diff
@@ -5,16 +5,6 @@ import * as bcrypt from "https://deno.land/x/bcrypt/mod.ts";
 import { v4 } from "https://deno.land/std/uuid/mod.ts";
 import { Client } from "https://deno.land/x/postgres@v0.11.3/mod.ts";
 import { config } from "https://deno.land/x/dotenv/mod.ts";
-// //============================================================
```

Comment:
Do we need this comments? Code for our node server

Reply:
I've removed this now